Formula/snapcast.rb

```diff
@@ -0,0 +1,51 @@
+class Snapcast < Formula
+  desc "Synchronous multi-room audio player"
+  homepage "https://github.com/badaix/snapcast"
+  url "https://github.com/badaix/snapcast/archive/v0.10.0.tar.gz"
+  version "0.0.10"
+  sha256 "e4a9e12aca8cfeed5e51866a0143e7e8d0a6e1ce0c076e8f78ce23c5c735c8d3"
+  revision 1
+  head do
+    url "https://github.com/badaix/snapcast.git"
+  end
+  depends_on "flac"
+  depends_on "libvorbis"
+  def install
+    args = %W[
+      prefix=#{prefix}
+      sysconfdir=#{etc}
+      CXX=#{ENV.cxx}
+    ]
+    if build.head?
+      system "git", "submodule", "update", "--depth", "1", "--init", "--recursive"
+    else
+      system "git", "clone", "--depth", "1", "https://github.com/badaix/popl.git", "externals/popl"
+      system "git", "clone", "--depth", "1", "https://github.com/chriskohlhoff/asio.git", "externals/asio"
+      system "git", "clone", "--depth", "1", "git://git.xiph.org/flac.git", "externals/flac"
+      system "git", "clone", "--depth", "1", "git://git.xiph.org/ogg.git", "externals/ogg"
+      system "git", "clone", "--depth", "1", "git://git.xiph.org/tremor.git", "externals/tremor"
+    end
+    system "make", "all", "TARGET=MACOS", *args
+    mkdir_p bin
+    bin.install "client/snapclient"
+    bin.install "server/snapserver"
+    mkdir_p man
+    cp_r "client/snapclient.1", man
```

Comment:
Please use `man1.install "client/snapclient.1"` instead of `cp_r` or `cp`. Install `*.1` stuff under `man1`; not `man`.